package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "compression": "1.7.2",
-    "express": "5.0.0-alpha.6",
+    "express": "4.16.3",
```

Comment:
Why using an older version?

Reply:
Mmmh&hellip; seems safer to stay away from alphas and prereleases until they're well tested?
That being said, I realise it was me who accepted that Greenkeeper update! :laughing:
Whatever you prefer.

Reply:
I prefer to stick with stable versions also